src/timeunit.ts

```diff
@@ -294,7 +294,8 @@ export function template(timeUnit: TimeUnit, field: string, shortTimeLabels: boo
    const template = '{{' + field + ' | time:\'' + out.join(' ') + '\'}}';
 
    // FIXME: Remove this RegExp Hack!!!
-   return template.replace(new RegExp('{{' + field + ' \\| time:\'\'}}', 'g'), '');
+   const escapedField = field.replace(/([.*+?^=!:${}()|\[\]\/\\])/g, '\\$1');
```

Comment:
I don't understand this line.  Can you add comment explaining what it does? 

Or even better, get rid of it.  I actually did not approve this RegExp first time I see it but it was already merged in master.  Thus I added this FIXME.